Ansible/ansible_collections/jfrog/platform/roles/artifactory/tasks/main.yml

```diff
@@ -1,3 +1,14 @@
+- name: Get artifactory_servers count
+  ansible.builtin.set_fact:
+    artifactory_server_count: "{{ groups['artifactory_servers'] | length }}"
```

Comment:
@bbaassssiiee this logic can be moved to defaults.yaml file

Reply:
I notice a variable `artifactory_ha_enabled` in defaults.yml, looks like a candidate for this logic...